src/Param.h

```diff
@@ -22,6 +22,20 @@ struct DomainSize
 	double height_;
 };
 
+struct InitialInfections
```

Comment:
I think this should be put into a different file.

Reply:
So do I but only when the code base is more modular